src/core_write.cpp

```diff
@@ -189,11 +190,11 @@ void OutputToJSON(uint256 &txid, int i,
         {
             CTxOutCT *s = (CTxOutCT*) baseOut;
             entry.pushKV("type", "blind");
-            entry.pushKV("valueCommitment", HexStr(&s->commitment.data[0], &s->commitment.data[0]+33));
+            entry.pushKV("ephemeral_pubkey", HexStr(s->vData.begin(), s->vData.end()));
             UniValue o(UniValue::VOBJ);
             ScriptPubKeyToUniv(s->scriptPubKey, o, true);
             entry.pushKV("scriptPubKey", o);
-            entry.pushKV("data_hex", HexStr(s->vData.begin(), s->vData.end()));
+            entry.pushKV("valueCommitment", HexStr(&s->commitment.data[0], &s->commitment.data[0]+33));
```

Comment:
We will want to make 33 at const as some point?

Reply:
a const for 33 is added as part of #696